source/components-and-apis.rst

```diff
@@ -215,6 +199,11 @@ See more about how using API in :ref:`Using API interface`.
      - /socket.io
      - `API - data-broker`_
      - No
+   * - X.509 Identity Management
+     - /x509
+     - /api
```

Comment:
/x509/v1?

Reply:
Currently it is `/api/v1` but I think I will end up changing to be in the same mold as kafka-ws (next release).